admin_guide/image_pull_secrets.adoc

```diff
@@ -0,0 +1,46 @@
+= Image Pull Secrets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+Docker registries can be secured to prevent unauthorized parties from accessing certain images.  If you are using the OpenShift docker registry and are pulling images for pods located in the same project, then your service account should already have permissions and no additional action should be required.  If this is not the case, then additional configuration steps are required.
+
+=== OpenShift Docker Registry Authentication and Authorization
+The OpenShift Docker Registry authenticates using the same tokens as the OpenShift API.  To perform a `docker login` against the OpenShift Docker registry, you can choose any username and email, but the password must be a valid OpenShift token.  In order to pull an image, the authenticated user must have `get` rights on the requested `imagestreams/layer`.  In order to push an image, the authenticated user must have `update` rights on the requested `imagestreams/layer`.
+
+By default, all service accounts in a project have rights to pull any image in the same project and the builder service account has rights to push any image in the same project.
+
+=== Allowing Pods in ProjectA to reference images from ProjectB
```

Comment:
globally, use valid project names in examples (lowercase, alphanumeric with hyphen separators)